doc/man1/cms.pod

```diff
@@ -560,7 +560,7 @@ The B<-debug_decrypt> option can be used to disable the MMA attack protection
 and return an error if no recipient can be found: this option should be used
 with caution. For a fuller description see L<CMS_decrypt(3)>).
 
-=head1 CAdES Basic Electronic Signature (CAdES-BES)
+=head1 CADES BASIC ELECTRONIC SIGNATURE (CADES-BES)
```

Comment:
I am not sure if the 'd' in 'CADES' should be also uppercased. 'CAdES' sounds like a proper noun from my limited English knowledges...

Reply:
I know, I flip-flopped and went for consistency.

Reply:
I believe the man-pages get head1s uppercased anyway. Maybe not html, though....

Reply:
Okay